conda-recipes/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "scvi" %}
+{% set version = "0.1.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "ca87155662d85605f86c5e86b7b9f64d881b882177b9642fff8f0ea215c6cb1a" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  # script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip >=9.0.1
+    - flake8 ==3.5.0
+    - coverage ==4.5.1
+    - sphinx ==1.7.1
+    - pytest ==3.4.2
+    - pytest-runner ==2.11.1
+    - numpy ==1.14.0
+    - pytorch ==0.4.0
+    - matplotlib ==2.2.2
+    - scikit-learn ==0.19.1
+    - scipy ==1.0.0
+    - h5py
+    - codecov
+    - pandas ==0.23.0
+    - loompy ==2.0.9
+  run:
+    - python
+    - pip >=9.0.1
+    - flake8 ==3.5.0
+    - coverage ==4.5.1
+    - sphinx ==1.7.1
+    - pytest ==3.4.2
+    - pytest-runner ==2.11.1
+    - numpy ==1.14.0
+    - pytorch ==0.4
+    - matplotlib ==2.2.2
+    - scikit-learn ==0.19.1
+    - scipy ==1.0.0
+    - h5py
+    - codecov
+    - pandas ==0.23.0
+    - loompy ==2.0.9
```

Comment:
Just the minimal requirements go here.

Reply:
The build errors if I use minimal requirement file :(. I suspect the error comes from the inconsistency between `requirements.txt` and version `0.1.1`'s `setup.py`, since `conda` package is built from `pip`, and `0.1.2` hasn't been uploaded on `pip`. 

I think I can either 
1) Use the current `meta.yml` to upload `0.1.1` to `conda`, or 
2) Upload the current `scvi` as `version 0.1.2` to `pip`, and then try building the new version with minimal requirements for `conda`. (I've put the minimal requirements into a separate folder locally for now).

Reply:
Uploading 0.1.2 to pip is fine